mean/modules/survey/views/p17_30.html

```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script src="https://code.angularjs.org/1.4.3/angular.js"></script>
+    <link rel="stylesheet" href="survey.css" />
+    <script src="gen%20info%20JS.js"></script>
+  </head>
+
+<!-- Insert header here -->
+    <body>
+        <h3>EXAMINATION</h3>
+        <hr>
+        <table class="info">
+            <tr>
+                <td class="col-md-1">
+                  <h4>RESPIRATORY</h4>
+                </td>
+                <td class="col-md-2">
+                  <input type="checkbox" name="respiratory_na" />NA or Unable to assess
```

Comment:
Are you doing anything with the input names?  How are you binding the information?